libsnark/polynomial_commitments/kzg10_batched.tcc

```diff
@@ -147,33 +147,42 @@ typename kzg10_batched_2_point<ppT>::evaluation_witness kzg10_batched_2_point<
     // For convenience of variable naming, let $f_i \in fs$ and $g_i in gs$ be
     // the two sets of polynomials. These are denoted $f_i$ and $f'_i$ in
     // [GWC19]. Similarly, $h_1$ and $h_2$ are used in place of $h$ and
-    // $h' in the paper, and $\gamma_1$ and $\gamma_2$ in place of
+    // $h'$ in the paper, and $\gamma_1$ and $\gamma_2$ in place of
     // $\gamma$ and $\gamma'$.
     //
-    // Compute:
+    // To generate the witness we must compute:
     //
-    //   h_1(X) = \sum_{i=1}^{t_1} \gamma_1^{i-1} (f_i(X) - f_i(z_1)/(X - z_1)
+    //   h_1(X) = \sum_{i=1}^{t_1} \gamma_1^{i-1} (f_i(X) - f_i(z_1))/(X - z_1)
     //
-    //   h_2(X) = \sum_{i=1}^{t_2} \gamma_2^{i-1} (g_i(X) - g_i(z_1)/(X - z_1)
+    //   h_2(X) = \sum_{i=1}^{t_2} \gamma_2^{i-1} (g_i(X) - g_i(z_1))/(X - z_2)
```

Comment:
g_i(z_1) -> should be z_2 instead of z_1

Reply:
this should be fixed in the latest version.  thanks.